projects/aca-playwright-shared/src/page-objects/components/adf-info-drawer.component.ts

```diff
@@ -36,4 +36,32 @@ export class AdfInfoDrawerComponent extends BaseComponent {
   public getIdField = (labelText: string) => this.getChild('[data-automation-id="library-id-properties-wrapper"]', { hasText: labelText });
   public getVisibilityField = (labelText: string) => this.getChild('[data-automation-id="library-visibility-properties-wrapper"]', { hasText: labelText });
   public getDescriptionField = this.getChild('[data-automation-id="library-description-properties-wrapper"] textarea');
+  public propertiesTab = this.getChild('.mat-tab-label-content').nth(0);
```

Comment:
I would consider using something else than Material classes as they might be different after Angular 15 upgrade which will break those tests

Reply:
Will be fixed in the next push